docs/updating_guide.md

```diff
@@ -0,0 +1,133 @@
+```
+
+#### Modify the Deployment Manifest
+
+- Before you begin any changes, please check the latest format of deployment manifest [here](https://seed-farmer.readthedocs.io/en/latest/manifests.html#deployment-manifest), where you can find out information about the supported attributes
+- Add required `toolchainRegion`
+- Add required `targetAccountMappings` with at least one account
+  - Mark one account with `default: true` and one region with `default: true`
+  - add a `.env` file at the root of the addf project which has the values of global kind of variables. In the below deployment (2nd) snippets, we declare PRIMARY_ACCOUNT as an env variable, and set its value in the .env file as `PRIMARY_ACCOUNT=1234567890`
+- Below is a snippet of how an existing deployment.yaml file would look like
+
+```yaml
+name: demo
+dockerCredentialsSecret: aws-addf-docker-credentials
+groups:
+  - name: optionals
+    path: manifests/demo/optional-modules.yaml
+  - name: core
+    path: manifests/demo/core-modules.yaml
+  - name: rosbag
+    path: manifests/demo/rosbag-modules.yaml
+```
+
+- Below is a snippet of how a deployment.yaml file would look like to support the 1.x version of seedfarmer
+
+```yaml
+name: demo
+toolchainRegion: us-west-2
+groups:
+  - name: optionals
+    path: manifests/example-dev/optional-modules.yaml
+  - name: core
+    path: manifests/example-dev/core-modules.yaml
+  - name: rosbag
+    path: manifests/example-dev/rosbag-modules.yaml
+targetAccountMappings:
+  - alias: primary
+    accountId:
+      valueFrom:
+        envVariable: PRIMARY_ACCOUNT
+    default: true
+    parametersGlobal:
+      dockerCredentialsSecret: aws-addf-docker-credentials
+    regionMappings:
+      - region: us-west-2
+        default: true
+```
+
+> The above snippets assume that you are deploying to `us-west-2` region
+
+#### Bootstrapping
+
+#### Setting the AWS Region
+
+ADDF submits build information to AWS CodeBuild via AWS CodeSeeder.  The initial submittal is done via AWS CLI, leveraging the configured active AWS profile.  If you would like to deploy ADDF to a region other than the region configured for the profile, you can explicitly set the region via:
+
+```bash
+export AWS_DEFAULT_REGION=<region-name>   (ex. us-east-1, eu-central-1)
+```
+
+Please see [HERE for details](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
+
+#### Bootstrap the CDKV2
+
+We use AWS CDK V2 as the standard CDK version because CDK V1 is set to maintenance mode beginning June 1, 2022. You  would need to bootstrap the CDK environment (one time per region) with V2 - see [HERE](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html).
+
+```bash
+cdk bootstrap aws://<account>/<region>
+```
+
+#### Bootstrap the Seedfarmer
+
+Bootstrap your existing account to convert to a Toolchain and a Target account using the below
+
+```sh
+seedfarmer bootstrap toolchain --project [project] --trusted-principals [principal-arns] --as-target
```

Comment:
in all of these examples, let's populate the `--project [project]` with `--project addf`